apitools/base/py/base_api.py

```diff
@@ -685,9 +696,14 @@ def _RunMethod(self, method_config, request, global_params=None,
             http = self.__client.http
             if upload and upload.bytes_http:
                 http = upload.bytes_http
+            opts = {
```

Comment:
To avoid putting the MakeRequest call in both branches of an if/else, since we don't want to explicitly pass `check_response_func=None`.